wp-content/themes/engage/assets/scss/components/_widget.scss

```diff
@@ -29,34 +29,59 @@
 		color: $black-grey;
 	}
 
-	.constantcontactwidget_form {
-		p {
-			margin-bottom: 1rem;
-			color: $black-grey;
-		}
+	.textwidget {
+		.wpcf7-form {
+			margin: 0;
 
-		input[type="text"] {
-			font: inherit;
-			margin-top: 1rem;
-			width: 100%;
+			.newsletter-prompt {
+				margin-bottom: 1.6rem;
+				color: $black-grey;
+			}
 
-			@include media($tablet) {
+			.newsletter-input {
+				font: inherit;
 				width: 100%;
+				border: 1px solid;
+
+				@include media($tablet) {
+					width: 100%;
+					border: none;
+				}
 			}
-		}
 
-		input[type="submit"] {
-			display: block;
-			background: $orange;
-			margin-top: 2.5rem;
-			margin-bottom: 1rem;
+			.newsletter-submit {
+				display: block;
+				width: 110px;
+				border-radius: 30px;
+				background: $orange;
```

Comment:
Are the `border-radius` and `background` props here because you can't add our existing button classes to the `.newsletter-submit` button? Ideally, we can just add our existing button classes to the `.newsletter-submit` rather than writing new CSS.

Reply:
The form overwrites the btn class, getting rid of all the border stuff and makes the width 100% automatically. So I can change the width to another % to fix it. The padding doesn't fix the problem